src/utils_format_graphite.c

```diff
@@ -120,7 +151,14 @@ static int gr_format_name_tagged(char *ret, int ret_len, value_list_t const *vl,
   if (postfix == NULL)
     postfix = "";
 
-  gr_copy_escape_part(n_host, vl->host, sizeof(n_host), escape_char, 1);
+  if (flags & GRAPHITE_REVERSE_HOST) {
+    int len_host = strlen(vl->host);
+    char r_host[len_host + 1];
```

Comment:
The size of `r_host` array can be `DATA_MAX_NAME_LEN` as this is the size of `vl->host`, so that computation of `len_host` is not necessary.